packages/playground/src/App.js

```diff
@@ -199,9 +199,9 @@ function App() {
   }, [handleUseMockBot]);
 
   // Feel free to change the text to whatever will help you dev faster
-  const handleCurrentCommandClick = useCallback(() => {
+  const handleCurrentCommandClick = () => {
```

Comment:
This line seems unrelated to the PR.

Reply:
Yeah it's just a correction for playground